pkg/operator/operator.go

```diff
@@ -221,7 +221,12 @@ func New(
 		mcoSecretInformer.Informer(),
 		ocSecretInformer.Informer(),
 		mcoCOInformer.Informer(),
-	} {
+	}
+
+	// this is for the FG
```

Comment:
we might need this @inesqyx otherwise clusters may fail trying to list this type? I am unsure

You'll need to make sure the MOSC and MOSB CRDs are installed correctly upon cluster installation or else this'll cause issues.